docs/src/details/xml/tools/cli.rst

```diff
@@ -1,23 +1,194 @@
-
 .. _editor_cli:
 
 ===
 CLI
 ===
 
+The LVGL Editor provides a Node-based command line tool, ``lved-cli.js``, for generating/compiling code from XML, validating projects, syncing from Figma, comparing trees, and running headless UI tests.
+
+.. note::
+   Pre-release build. Intended for development and CI. Expires end of 2025.
+
+.. toctree::
+   :maxdepth: 2
+   :local:
+
+
 Overview
 ********
 
+- **Binary:** ``lved-cli.js`` (Node script)
+- **Platform:** Node 18+ recommended
```

Comment:
If it's a requirement, please describe a basic installation guide on Linux  and a link to a how-to-install website.